react/__tests__/components/__snapshots__/InfoCard.test.js.snap

```diff
@@ -16,7 +16,7 @@ exports[`<InfoCard /> should insert span with class on headline 1`] = `
         <div
           style="display: contents;"
         >
-          HEADLINE 
+          HEADLINE
```

Comment:
The tests depend on this whitespace to work. It's not a good thing, but since it's out of the scope of this PR, let's keep it as it is, for now.

```suggestion
          HEADLINE 
```